besca/tl/auto_annot/__init__.py

```diff
@@ -1,3 +1,7 @@
-from ._auto_annot import read_data, read_raw, read_adata, merge_data, naive_merge, scanorama_merge, remove_genes, intersect_genes, remove_nonshared, fit, linear_svm, rbf_svm, sgd_svm, random_forest, logistic_regression, logistic_regression_ovr, logistic_regression_elastic, adata_predict, predict, report,  plot_confusion_matrix, adata_pred_prob, predict_proba
+from ._auto_annot import read_data, read_raw, read_adata, merge_data, naive_merge, \
+        scanorama_merge, remove_genes, intersect_genes, remove_nonshared, fit, linear_svm, \
+        rbf_svm, sgd_svm, random_forest, logistic_regression, logistic_regression_ovr, logistic_regression_elastic, adata_predict, predict,  adata_pred_prob, predict_proba
 
```

Comment:
Instead of removing `report()`, I would make it calling the new function and comment that it is DEPRECATED, to ensure backwards-combability with older auto-annot notebooks.